src/main/java/org/folio/auth/authtokenmodule/tokens/TokenValidationException.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.auth.authtokenmodule.tokens;
+
+/**
+ * When a token is provided and it hasn't passed validation, for any reason,
+ * a TokenValidationException will be present. Token validation exceptions have
+ * an opinion about what http response code should be returned to clients.
+ */
+public class TokenValidationException extends Exception {
+  public int httpResponseCode;
+
+  public TokenValidationException(String message, int responseCode) {
+    super(message);
+    httpResponseCode = responseCode;
```

Comment:
Can you rename `responseCode` to `httpResponseCode` and use `this.httpResponseCode = httpResponseCode` in this and next method?